README.md

```diff
@@ -1,18 +1,28 @@
 # `hardware.inc`
 ### Gameboy Hardware definitions
-`hardware.inc` has been the standard include file containing Game Boy hardware definitions for use in [rgbds](https://github.com/gbdev/rgbds) projects for over 20 years.
+`hardware.inc` has been the standard include file containing Game Boy hardware definitions for use in [RGBDS](https://rgbds.gbdev.io) projects for over 20 years.
 
 The file was originally created by Jeff Frohwein in 1997, who still hosts [his latest version (2.3)](http://www.devrs.com/gb/files/hardware.zip) on his great [Dev'rs](http://devrs.com) website.
 
 Although Jeff tried to version control the file with a rudimentary change log at the top of the file, people have added small changes throughout the years, often without bumping the version number.
 
-This repo is an attempt at version control, using [@AntonioND](http://github.com/AntonioND) fork as baseline.
+This repo has become the new official reference for `hardware.inc`, using [@AntonioND](http://github.com/AntonioND)'s fork as the baseline.
+
+## Contributing
+
+This repository's `master` branch should be considered production.
+That is, each commit represents a new release, which thus needs the embedded version number bumped (and an entry added to the changelog at the beginning of the file).
+
+We follow [semantic versioning](https://semver.org).
```

Comment:
```suggestion
We follow [semantic versioning](https://semver.org),
```

Reply:
Wouldn't the period be more grammatically correct in this case?

Reply:
A semicolon, actually.

Reply:
Updated to be the semicolon instead as per your suggestion, ISSO.